nBTC/sources/nbtc.move

```diff
@@ -147,6 +148,21 @@ public struct RedeemInactiveDepositEvent has copy, drop {
     amount: u64, // in satoshi
 }
 
+public struct RedeemRequestCreatedEvent has copy, drop {
+    redeem_id: u64,
+    redeemer: address,
+    recipient_script: vector<u8>,
```

Comment:
let's add a comment to clarify what it is: bitcoin address? Full script pubkey?

Reply:
This is a full pubkey/lockscript.